### PR TITLE
#FN-42 feat: Seller create/add items

### PR DESCRIPTION
**Pull request by Lydie260**
# PR Description
As a Seller
I should be able to add/create items for sale
This page should only be accessible to sellers, i.e authenticated
Such that the stock can be seen by any requesting client
Handle all responses and errors from the backend.

Please include a brief description of the pull request you want to raise.

## Description of tasks that were expected to be completed

List all the tasks and sub-tasks that were assigned to you Make sure you check every completed task. list them in this style

- [x] Add product to the seller's collection

## Description of tasks that were expected to be completed and that are not yet completed

- [x] Add new product
- [x] Display new added product

## How this may be tested?
Please describe the tests that you ran to verify your functionalities. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

0. clone the project on https://github.com/atlp-rwanda/ecommerce-app-cogito-fe.git
1. run `yarn i` to install all required dependencies
2. run `yarn start`
3. Go to the login page for authentication or create a new account if you don't have one N.B login as a seller.
4. After logging as a seller you will be directed to seller's page. 
5. After accessing the seller's page click on add new product fill out the form then click add.

## Screenshots (If appropriate/applicable)

Provide screenshots related to your PR in case it's appropriate and applicable

## Please check this Checklist before you submit your PR:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] My code generates no warnings
- [x] There are no vulnerabilities

### CODE REVIEW

**Review by Honore123 — COMMENTED** [review]
Allow vendors to add at least 4 images per product

**Review by Honore123 — APPROVED** [review]

**Review by Charlesntwari — APPROVED** [review]

**Review by dushimeemma — CHANGES_REQUESTED** [review]
- Why are you using `npm` while others are using `yarn`
- Enable to test your PR, what are the steps to follow?

**Comment by Lydie260** [comment]
> * Why are you using `npm` while others are using `yarn`
> * Enable to test your PR, what are the steps to follow?
Done with correction

File: src/App.tsx [hunk 1]
```diff
@@ -6,18 +6,23 @@ import SignupPage from './components/register';
...
 
 const App: React.FC = () => {
   return (
     <>
+      <div></div>
+
       <div>
         <Router>
-          <NavBar />
+          {/* <NavBar /> */}
```
Comment (dushimeemma) [inline src/App.tsx]: Why did you comment on this line of code?

File: src/App.tsx [hunk 2]
```diff
@@ -28,10 +33,13 @@ const App: React.FC = () => {
             <Route path="/UpdatePassword" element={<UpdatePassword />} />;
             <Route path="/profile" element={<ProfileView />} />
             <Route path="/editProfile" element={<EditProfileView />} />
-            <Route path= "/login/success" element={<LoginSuccess />} />;
+            <Route path="/login/success" element={<LoginSuccess />} />;
+            <Route path="/login/success" element={<LoginSuccess />} />;
+            <Route path="/sellerPage" element={<VendorPage />} />
           </Routes>
-          <Footer />
+          {/* <Footer /> */}
```
Comment (dushimeemma) [inline src/App.tsx]: Why did you comment this line of code?

**Review by dushimeemma — CHANGES_REQUESTED** [review]

File: src/App.tsx (hunk 1 above)
Comment (Lydie260) [reply src/App.tsx]: I did it because for dashboard it looks weird to have dashboard and footer

File: src/App.tsx (hunk 2 above)
Comment (Lydie260) [reply src/App.tsx]: I did it because for dashboard it looks weird to have dashboard and footer

**Review by dushimeemma — APPROVED** [review]

**Pull request merged**